src/Components/Members/index.js

```diff
@@ -1,30 +1,25 @@
 import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { getMembers, deleteMember } from '../../redux/members/thunks';
+
 import styles from './members.module.css';
 import { SharedTable, Button } from '../Shared';
 
 function Members(props) {
+  const [loading, setLoading] = useState(true);
   const [members, setMembers] = useState([]);
-
-  const getMembs = async () => {
-    const response = await fetch(`${process.env.REACT_APP_API_URL}/api/member`);
-    const data = await response.json();
-    data.data.forEach((item) => {
-      item.birthDate = item.birthDate.substring(0, 10);
-    });
-    setMembers(data.data);
-  };
+  const data = useSelector((state) => state.members);
+  const dispatch = useDispatch();
 
   useEffect(() => {
-    getMembs();
-  }, []);
+    dispatch(getMembers(setLoading));
+    setMembers(data.list);
+  }, [members]);
 
-  const deleteMemb = async (id) => {
-    try {
-      await fetch(`${process.env.REACT_APP_API_URL}/api/member/${id}`, { method: 'DELETE' });
-      setMembers(members.filter((member) => member._id !== id));
-    } catch (error) {
-      alert(error.message);
-    }
+  const delMember = (id) => {
+    dispatch(deleteMember(id));
+    setMembers(members.filter((member) => member._id !== id));
   };
 
   return (
```

Comment:
i think we are missing the success message for the delete of a member

Reply:
Solved